spot-client/src/common/backend/analyticsEvents.js

```diff
@@ -0,0 +1,11 @@
+export const permanentPairingCodeEvents = {
+    /**
+     * When backend pairing is lost.
+     */
+    PAIRING_FAIL: 'permanent-pairing-code-fail',
+
+    /**
+     * When backend pairing is established.
+     */
+    PAIRING_SUCCESS: 'permanent-pairing-code-success'
+};
```

Comment:
I thought all analytics events were going to be in files within /analytics directories. Should /backend be concerned with analytics? I see below you added analytics-related middleware to backend as well.

Reply:
I can have this moved to common/analytics

Reply:
That's what I'm asking about, yes. Is there an explicit reason why you moved away from having all analytics live in /analytics directories?

Reply:
No, I will move it there